### PR TITLE
Edgaro-vardas-main

### PR DESCRIPTION
**Pull request by ImNotHereForToday**
Cia mano vardas :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,13 @@
         <!-- Edmundas -->
     </header>
     <main>
-        <!-- Edgaras -->
+        <h3>Labas</h3>
+        <h3>,</h3>
+        <h3>mano</h3>
+        <h3>vardas</h3>
+        <h1>EDGARAS</h1>
+        <h1>¯\_(ツ)_/¯</h1>
```
Comment (KristijonasTar) [inline index.html:18]: :)

Comment (ImNotHereForToday) [reply index.html:18]: good

**Review by KristijonasTar — APPROVED** [review]
Looks good

File: index.html (hunk 1 above)
Comment (EdmundasKazakevicius) [inline index.html:18]: aaa

**Review by EdmundasKazakevicius — APPROVED** [review]

**Pull request merged**